stellarphot/table_representations.py

```diff
@@ -35,8 +37,83 @@ def _constructor(loader, node):
     AstropyLoader.add_constructor(class_string, _constructor)
 
 
-# This code is deliberately executable so that it can be executed on import, which
-# should assure that Table representations are generated for all models.
-for model_name in models.__all__:
-    model_class = getattr(models, model_name)
-    generate_table_representers(model_class)
+def serialize_models_in_table_meta(table_meta):
+    """
+    Serialize the models in the table metadata **IN PLACE**.
+    This is used to ensure that the models are represented as simple
+    dictionaries when written to disk.
+
+    Parameters
+    ----------
+    table : `~astropy.table.Table`
+        The table whose metadata will be serialized.
+    """
+    model_classes = tuple(getattr(models, model_name) for model_name in models.__all__)
+
+    for key, value in table_meta.items():
+        # If the value is a model instance, serialize it
+        if isinstance(value, model_classes):
+            model_instance = value
+            # So, funny story. model_dump gives you a nice dictionary, in which
```

Comment:
What data are we storing in the metadata for the ecsv files? This is more so as a reminder for myself and not so much to do with any issues I see in this pull request. I am assuming its all the data in the photometry settings json.